doc/common/README.md

```diff
@@ -569,14 +569,38 @@ d. Участники, проверяющие код, добавляют ком
 
 e. Комментарии отмечает выполненными автор изменений, код которого просматирвают.
 
-f. После корректировки замечаний и реализации предложений должен запрашиваться
+f. В начале процесса исправления замечаний задача переводится в соотвествующий рабочий статус. 
+   После завершения исправлений задача переводится обратно в статус ревью.
+
+g. После корректировки замечаний и реализации предложений должен запрашиваться
 повторный обзор кода.
 
-g. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
+h. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
 использовать такой способ коммуникации, при котором ответ затягивается, а соответственно
 затягивается и ревью.
 
-#### 6.4.4. Противоречия и споры в процессе обзора кода
+#### 6.4.4. Процесс обзора задачи, не связанных с системой управления кодом (Github, Gitlab)
+
+Такие задачи, как документирование в Notion, Confluence и пр. 
+
+1. Ревьюер в процессе обзора кода фиксирует свои замечания и пожелания в одном 
+месте. \
+Например:
+[Документ GoogleDocs](https://docs.google.com/spreadsheets/d/1yZT4qOec28r-uTDg9uz3TOvNi5I0ePbppZa7URWby1s/edit?usp=sharing)
```

Comment:
We also can use slack one of the following ways:
1. Comments to task in review channel. One comment - one message as we did before. Then author of changes mark each item with green checkmark.
2. Find special app for slack with checkboxes if we want checkboxes :)

Reply:
done